lang/node-mozilla-iot-gateway/Makefile

```diff
@@ -0,0 +1,78 @@
+#
+# Copyright (C) 2018 OpenWrt.org
```

Comment:
no. you don't have and can't do copyright assignment to openwrt like this.  Either drop it entirely, or put your own name there.